pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -169,6 +170,7 @@ func BuildStorageFactory(masterConfig configapi.MasterConfig, server *kapiserver
 	if err != nil {
 		return nil, err
 	}
+	storageGroupsToEncodingVersion[autoscaling.GroupName] = kubecompatextensionsv1beta1.SchemeGroupVersion
```

Comment:
Why this? Shouldn't we store them in the new format and enforce migration, and then in 1.7 finally remove extensionsv1beta1.HPA support?